spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -2108,26 +2116,69 @@ See {spring-batch-api}/core/configuration/annotation/EnableBatchProcessing.html[
 
 For more about Spring Batch, see the {spring-batch}[Spring Batch project page].
 
-
-
-[[howto-execute-spring-batch-jobs-on-startup]]
-=== Execute Spring Batch Jobs on Startup
+[[howto-run-spring-batch-jobs-on-startup]]
```

Comment:
If the anchor is changed, I'd rather harmonize it to the same prefix as you've done for everything else.

Reply:
Got it.